### PR TITLE
docs(ngrx-router): clarify that the extracted variables are functions

### PR DESCRIPTION
**Pull request by maartentibau**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngrx/platform/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Documentation has been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Closes #

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by maartentibau** [comment]
I tried to add this as a note in my first try, but I removed it again. But indeed adding an example like this would make even more sense.

**Comment by maartentibau** [comment]
Changed the PR code according to your remarks @timdeschryver

File: projects/ngrx.io/content/guide/router-store/selectors.md [hunk 1]
```diff
@@ -32,6 +32,20 @@ const {
...
+   (cars, carId) => {
+       return cars[carId];
+    }
+)
+
+export const selectCarsByColor = createSelector(
+   selectCarEntities,
+   selectQueryParams,
+   (cars, params) => {
+       return cars.filter(c => c.color === params['color'];
```
Comment (timdeschryver) [inline projects/ngrx.io/content/guide/router-store/selectors.md]: There's a missing closing bracket here 👀

```
cars.filter(c => c.color === params['color']);
```

Comment (maartentibau) [reply projects/ngrx.io/content/guide/router-store/selectors.md]: https://github.com/ngrx/platform/pull/2289

**Pull request merged**